docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -230,6 +232,59 @@ jobs:
 
 ```
 
+### `SEMGREP_REPO_DISPLAY_NAME`
+
+Set `SEMGREP_REPO_DISPLAY_NAME` to define the name used for the project associated with a given Semgrep scan. By default, `SEMGREP_REPO_DISPLAY_NAME` has the same value as `SEMGREP_REPO_NAME`. This allows you to use a different name for your project than the repository name or to scan a monorepo as multiple projects.
+
+#### Scanning a monorepo as multiple projects
```

Comment:
```suggestion
#### Scan a monorepo as multiple projects
```
We use simple present these days for the header, mostly to be clear when something is a verb and something is a noun.